oop_rock_paper_scissors.py

```diff
@@ -0,0 +1,161 @@
+""" Object-Oriented Rock Paper Scissors """
+
+import random
+
+
+class Move(object):
+
+    def __eq__(self, other):
+        if type(self) == type(other):
+            return True
+        return False
+
+
+class Scissors(Move):
+
+    def __gt__(self, other):
+        if isinstance(other, Paper):
+            return True
+        return False
+
+    def __lt__(self, other):
+        if isinstance(other, Rock):
+            return True
+        return False
+
+
+class Paper(Move):
+
+    def __gt__(self, other):
+        if isinstance(other, Rock):
+            return True
+        return False
+
+    def __lt__(self, other):
+        if isinstance(other, Scissors):
+            return True
+        return False
+
+
+class Rock(Move):
+
+    def __gt__(self, other):
+        if isinstance(other, Scissors):
+            return True
+        return False
+
+    def __lt__(self, other):
+        if isinstance(other, Paper):
+            return True
+        return False
+
+
+class User(object):
+
+    def __init__(self):
+        self.score = 0
+        self.choice = None
+        self.instance = None
+
+    def get_move(self):
+        user_choice = input('Choose rock, paper, or scissors: ')
+        self.choice = user_choice
+
+    def play_move(self):
+        return self.choice
+
+    # function to convert input to Move instance
+    def convert_input_to_move(self):
+        if self.choice == 'rock':
+            self.instance = Rock()
+        elif self.choice == 'paper':
+            self.instance = Paper()
+        elif self.choice == 'scissors':
+            self.instance = Scissors()
+        else:
+            print('Invalid Input')
+            exit()
+
+    def get_move_instance(self):
+        return self.instance
+
+
+class NPCUser(User):
+
+    def get_move(self):
+        computer_choice = random.choice(('rock', 'paper', 'scissors'))
+        self.choice = computer_choice
+
+
+class Game(object):
+
+    def __init__(self):
+        self.rounds = 0
+
+    # check for odd number of rounds
+    def get_rounds(self):
+        rounds = int(input('How many odd rounds would you like to play? '))
+        self.rounds = rounds
+
+    def return_rounds(self):
```

Comment:
What's the purpose of `return_rounds` as a method?  Perhaps it would be more appropriate for `main` to simply access `game.rounds` instead of accessing via a method?